### PR TITLE
Add rails4 end to end test

### PR DESCRIPTION
**Pull request by kbaum**
End to end test now runs for rails5 and rails4 within travis. Using a 2nd rails4 Gemfile on travis that depends on rails4 instead of rails5.

### CODE REVIEW

File: coverband.gemspec [hunk 1]
```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'redis'
   spec.add_development_dependency 'test-unit'
-  spec.add_development_dependency 'rails', '~>5'
```
Comment (danmayer) [inline coverband.gemspec]: with two Gemfiles which explicitly call out Rails, do we still need or want this? @kbaum

**Comment by danmayer** [comment]
added one comment and how does one run the two different Rails tests could you perhaps add notes on how to do that to the readme? @kbaum

File: coverband.gemspec (hunk 1 above)
Comment (kbaum) [reply coverband.gemspec]: Oh.. good catch! We don't need this in the gemspec anymore.

**Comment by kbaum** [comment]
Will do. rails5 is the in the default Gemfile so it's just:

```
bundle exec rake test
```

For rails4:

```
BUNDLE_GEMFILE=Gemfile.rails4 bundle exec rake test
```

**Comment by kbaum** [comment]
Added to README. Let me know what you think.
thx.

**Comment by danmayer** [comment]
nice +1 very cool to have the end to end tests in should help with tests verifying Railsties integration work.

**Pull request merged**